modules/keepalived-limitation-for-ipi.adoc

```diff
@@ -0,0 +1,48 @@
+[discrete]
+[id="keepalived-limitation-for-ipi_{context}"]
+= Identifying pre-existing virtual routers
+
+With installer-provisioned infrastructure (IPI) installations, the Virtual IPs are managed using multicast (VRRPv2 or VRRPv3), and there is a limitation of 255 unique virtual routers per multicast domain. Each VIP has a virtual router id associated with it based on the cluster name. Duplication of these ids can result in a conflict, preventing the Virtual IPs from being assigned to the node. Therefore, if you have pre-existing virtual routers using the standard IPv4 or IPv6 multicast groups, you should check to see which VIPs the installation will choose to avoid conflicts.
+
+To discover the VIPs that will be chosen, you can use the runtimecfg utility.
+
+.Procedure
+
+. Run the runtimecfg utility with a domain name, in this example `cnf10`:
++
+[source, terminal]
+----
+$ podman run quay.io/openshift/origin-baremetal-runtimecfg:TAG vr-ids cnf10
+----
++
+.Example output
+[source, terminal]
+----
+APIVirtualRouterID: 147
+DNSVirtualRouterID: 158
+IngressVirtualRouterID: 2
+----
++
+Where TAG is the release you are going to install, for example, 4.5.
++
+. Run the utility again using a different domain name, in this example `cnf11`:
++
+[source, terminal]
+----
+$ podman run quay.io/openshift/origin-baremetal-runtimecfg:TAG vr-ids cnf11
```

Comment:
We need to take it from the release payload:
https://bugzilla.redhat.com/show_bug.cgi?id=1886450#c7

Reply:
Done.